src/site/content/en/blog/top-cwv-2023/index.md

```diff
@@ -0,0 +1,203 @@
+
+Despite this, [a significant number of websites](https://almanac.httparchive.org/en/2022/performance#bfcache-eligibility) are ineligible for the bfcache and so are missing out on this free web performance win for a significant number of navigations. Unless your page is loading sensitive information that you don't want to be restored from memory, you'll want to make sure that your pages are eligible.
+
+Site owners should check that their pages are [eligible for the bfcache](/bfcache/#optimize-your-pages-for-bfcache) and work on any reasons why they are not. Chrome already [has a bfcache tester in DevTools](/bfcache/#test-to-ensure-your-pages-are-cacheable) and this year we plan to enhance tooling here with [a new Lighthouse audit performing a similar test](https://github.com/GoogleChrome/lighthouse/issues/13960) and [an API to measure this in the field](https://chromestatus.com/feature/5684908759449600).
+
+While we have included the bfcache in the CLS section, as we saw the biggest gains there so far, the bfcache will also improve the other Core Web Vitals, i.e. with 0 LCP and much improved FID and INP. It is one of [a number of instant navigations](https://calendar.perfplanet.com/2022/fast-is-good-instant-is-better/) available to drastically improve page navigations.
+
+### Avoid animations/transitions that use layout-inducing CSS properties
+
+Another common source of layout shifts is when elements are animated. For example, cookie banners or other notification banners can push content out of their way, and will cause a layout shift if they do.
+
+While HTTP Archive data can't conclusively connect animations to layout shifts, the data does show that pages that animate any CSS property that *could* affect layout are 15% less likely to have "good" CLS than pages overall. Some properties are associated with worse CLS than others. For instance, pages that animate `margin` or `border` widths have "poor" CLS at almost twice the rate that pages are assessed as poor overall.
+
+If an element must change its location or the area it takes up, the easiest way to avoid these shifts is to pull the animations outside of the normal document flow. The most common way is to **use animations that don't affect layout**, like `transform`. Animating with `transform` can have the added benefit of [improving page performance overall](/animations-guide/) by moving more animation work onto the GPU and off the main thread. A page could also remove the element itself from the document flow, like with `position: absolute` or `fixed`, though animating some properties (like `top`) could still trigger a layout shift on the element itself.
```

Comment:
```suggestion
This is perhaps not surprising because any time you transition or animate *any* layout-inducing CSS property, it will result in [layout shifts](/cls/#layout-shifts-in-detail), and if those layout shifts are not within 500 milliseconds of a user interaction, they will impact CLS.

What may be surprising to some developers is that this is true even in cases where the element is taken outside of the normal document flow. For example, absolutely positioned elements that animate `top` or `left` will cause layout shifts, even if they aren't pushing other content around. However, if instead of animating `top` or `left` you animate `transform:translateX()` or `transform:translateY()`, it won't cause the browser to update page layout and thus won't produce any layout shifts.

Preferring animation of CSS properties that can be updated on the browser's compositor thread has long been [a performance best practice](/animations-guide/) because it moves that work onto the GPU and off the main thread. And in addition to it being a general performance best practice, it can also help improve CLS.

As a general rule, never animate or transition any CSS property that requires the browser to update the page layout, unless you're doing it in response to a user tap or key press (though [not `hover`](/cls/#user-initiated-layout-shifts)). And whenever possible, prefer transitions and animations using the CSS [`transform`](https://developer.mozilla.org/docs/Web/CSS/transform) property.
```

Reply:
Done